### PR TITLE
Grid css

### PR DESCRIPTION
**Pull request by fededirocco**
## Summary
* Implemented grid css in website.
* Modified tech scss with grid properties.
* Modified index html with grid items. 

## Trello Card
https://trello.com/c/YjHGoBwO/12-redise%C3%B1o-de-la-p%C3%A1gina

## Screenshots
image
image
image

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+.tech-grid-container {
+  display: grid;
+  grid-template-areas: 
+  'left main1 main2'
+  'left main3 main4';
+  grid-template-columns: repeat(3, 33%);
+  grid-template-rows: repeat(2, 50%);
+  margin: 0 100px 100px;
+
+  .first {
```
Comment (jjuli93) [inline scss/tech.scss]: you can put this styles in the item1 class

File: scss/tech.scss [hunk 2]
```diff
@@ -1,25 +1,41 @@
-.tech-container {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 10px 70px;
-  width: 100%;
+.tech-grid-container {
+  display: grid;
+  grid-template-areas:
```
Comment (FrankIglesias) [inline scss/tech.scss]: define this without grid areas. Just setting the first element the height of two rows

File: scss/tech.scss [hunk 3]
```diff
@@ -1,25 +1,41 @@
...
-  display: flex;
-  flex-wrap: wrap;
-  padding: 10px 70px;
-  width: 100%;
+.tech-grid-container {
+  display: grid;
+  grid-template-areas: 
+  'left main1 main2'
+  'left main3 main4';
+  grid-template-columns: repeat(3, 33%);
```
Comment (FrankIglesias) [inline scss/tech.scss]: use fr

File: scss/tech.scss [hunk 4]
```diff
@@ -1,25 +1,41 @@
...
+  grid-template-rows: repeat(2, 50%);
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss (hunk 2 above)
Comment (fededirocco) [reply scss/tech.scss]: I used grid-column-start, grid-column-end, grid-row-start and grid-row-end but I don't know if it's what you mean

File: index.html [hunk 5]
```diff
@@ -14,63 +14,63 @@
...
-    <div class="box">
+    </div>
+    <div class="item2 box">
       <img class="logo" src="img/angular_icon.svg" alt="Angular logo" />
       <h2 class="name">AngularJS</h2>
       <h3 class="release">Release: 2010</h3>
       <p class="desc">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.</p>     
     </div>
-    <div class="box">
+    <div class="item3 box">
```
Comment (FrankIglesias) [inline index.html]: Now you can delete this items

**Pull request closed without merging**